### PR TITLE
Feat/clients

### PR DESCRIPTION
**Pull request by pyramation**

### CODE REVIEW

File: packages/ast/src/clients/rpc/query/__snapshots__/rpc.query.test.ts.snap [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    return promise.then(data => QueryParamsResponse.decode(new _m0.Reader(data)));
+  }
+
+  ModuleAccounts(request: QueryModuleAccountsRequest): Promise<QueryModuleAccountsResponse> {
+    const data = QueryModuleAccountsRequest.encode(request).finish();
+    const promise = this.rpc.request(\\"cosmos.auth.v1beta1.Query\\", \\"ModuleAccounts\\", data);
+    return promise.then(data => QueryModuleAccountsResponse.decode(new _m0.Reader(data)));
+  }
+
+  Bech32Prefix(request: Bech32PrefixRequest): Promise<Bech32PrefixResponse> {
```
Comment (pyramation) [inline packages/ast/src/clients/rpc/query/__snapshots__/rpc.query.test.ts.snap:71]: @assafmo I noticed that the version in cosmjs didn't' have `Bech32Prefix` and `AddressBytesToString` — I'm guessing this is still correct. Does it look ok to you?

Comment (assafmo) [reply packages/ast/src/clients/rpc/query/__snapshots__/rpc.query.test.ts.snap:71]: These queries are new in cosmos-sdk v0.46: https://github.com/cosmos/cosmos-sdk/blob/5019459b1b2028119c6ca1d80714caa7858c2076/proto/cosmos/auth/v1beta1/query.proto#L37-L64

We should probably have a registry of protobuf derived ts definitions for every tagged version of every chain, tendermint, ibc-go, cosmos-sdk, liquidity-staking-module, etc.

Comment (assafmo) [reply packages/ast/src/clients/rpc/query/__snapshots__/rpc.query.test.ts.snap:71]: Then decouple the client code from these definitions. Have the client devs import the definitions of the chain that they're targeting.

Comment (pyramation) [reply packages/ast/src/clients/rpc/query/__snapshots__/rpc.query.test.ts.snap:71]: great catch! ok so this is correct but it's for the 0.46. 

sounds like a good plan to support all the tagged versions. Now you got me thinking... in future we may have a situation where an app wants to use two chains, and one uses 0.45 and the other 0.46 and we need to support both in the same process.

Comment (pyramation) [reply packages/ast/src/clients/rpc/query/__snapshots__/rpc.query.test.ts.snap:71]: an example use case: keplr governance UI and a user needs to vote. One chain uses 0.45 and another 0.46, my guess is, they'd need to have both cosmos versions.

**Pull request merged**